hotsos/defs/events/openvswitch/ovs/ovs-vswitchd.yaml

```diff
@@ -17,3 +17,6 @@ inactivity-probe:
 involuntary-context-switches:
   expr: '([0-9-]+)T(\d+):\d+:\d+\.\S+Z.+\|timeval\|WARN\|context switches: 0 voluntary, (\d+) involuntary'
   hint: timeval
+assertion-failures:
+  expr: '([0-9-]+)T[0-9:\.]+Z.+\|util.+\|EMER\|\S+: assertion '
```

Comment:
as discussed with brian i was wrong in my statement above because search.constraints (i.e. the yaml property) are only used in scenario defs not events.